### DISCUSSION TITLE
How do you pass boolean options to protoc?

### DISCUSSION THREAD

**Discussion by roberttod** [timostamm/protobuf-ts#379] [Q&A]
```bash
npx protoc --ts_nocheck --ts_out ./proto --proto_path ../protobuf/ ../protobuf/**/*.proto
```

errors with `Missing value for flag: --ts_nocheck`

```bash
npx protoc --ts_nocheck true --ts_out ./proto --proto_path ../protobuf/ ../protobuf/**/*.proto
```
errors with `Unknown flag: --ts_nocheck`

It's not documented anywhere how to set these flags, and can't find anything in issues or discussions. Thanks!

**Comment by jcready** [comment]
It's mentioned in the [manual](https://github.com/timostamm/protobuf-ts/blob/master/MANUAL.md#the-protoc-plugin). You'd pass that as `--ts_opt ts_nocheck`. Multiple options can be specified with comma separation `--ts_opt ts_nocheck,eslint_disable`

```bash
npx protoc \
  --ts_opt ts_nocheck \
  --ts_out ./proto \
  --proto_path ../protobuf/ \
  ../protobuf/**/*.proto
```
